server/data_filter.py

```diff
@@ -21,7 +21,7 @@ def operations_callback(ops: defaultdict) -> None:
         # print all texts just as demo that data stream works
         post_with_images = isinstance(record.embed, models.AppBskyEmbedImages.Main)
         inlined_text = record.text.replace('\n', ' ')
-        logger.info(
+        logger.debug(
```

Comment:
the idea behind "info" here was about to show the user on the very first run that it works. i agree that we need an ability to easily turn it off. so let's change it to debug level as you did, but bump default logging level to DEBUG? and optionally make level configurable form the env vars?

Reply:
I'm thing there's got to be some way to check if the flask development server is running in debug mode. That way, we could keep the default log level ant infom but automatically switch the log level to debug if `flask --debug` is used.

Reply:
Done. Now the log level will switch from INFO to DEBUG when the flask CLI is used. That should also encourage people to switch to a WSGI server like waitress.